### PR TITLE
Mention COMPOSE_FILE and COMPOSE_PROJECT_NAME

### PR DESCRIPTION
**Pull request by Plopix**
https://github.com/symfony/cli/issues/323

### CODE REVIEW

**Comment by javiereguiluz** [comment]
@fabpot could you please review these proposed changes? Thanks!

File: setup/symfony_server.rst [hunk 1]
```diff
@@ -357,6 +357,25 @@ When Docker services are running, browse a page of your Symfony application and
 check the "Symfony Server" section in the web debug toolbar; you'll see that
 "Docker Compose" is "Up".
 
+If your Docker Compose file is not at the root of the project you can pass that information
```
Comment (fabpot) [inline setup/symfony_server.rst]: , you can pass

File: setup/symfony_server.rst [hunk 2]
```diff
@@ -357,6 +357,25 @@ When Docker services are running, browse a page of your Symfony application and
...
+to the Symfony Server. It works exactly as docker-compose.
```
Comment (fabpot) [inline setup/symfony_server.rst]: exactly the same as for ``docker-compose``

File: setup/symfony_server.rst [hunk 3]
```diff
@@ -357,6 +357,25 @@ When Docker services are running, browse a page of your Symfony application and
...
+
+Let's say you have your `docker-compose.yaml` file under a `docker/` directory.
```
Comment (fabpot) [inline setup/symfony_server.rst]: You need to use double ticks

**Review by fabpot — APPROVED** [review]

File: setup/symfony_server.rst (hunk 1 above)
Comment (Plopix) [reply setup/symfony_server.rst]: done

File: setup/symfony_server.rst (hunk 2 above)
Comment (Plopix) [reply setup/symfony_server.rst]: I put the double tick here too. Done

File: setup/symfony_server.rst (hunk 3 above)
Comment (Plopix) [reply setup/symfony_server.rst]: done

**Comment by Plopix** [comment]
I made the changes @fabpot. Thanks

File: setup/symfony_server.rst [hunk 4]
```diff
@@ -357,6 +357,25 @@ When Docker services are running, browse a page of your Symfony application and
 check the "Symfony Server" section in the web debug toolbar; you'll see that
 "Docker Compose" is "Up".
 
+If your Docker Compose file is not at the root of the project, you can pass that information
+to the Symfony Server. It works exactly the same as for ``docker-compose``.
+
+Let's say you have your ```docker-compose.yaml`` file under a ```docker/`` directory.
```
Comment (fabpot) [inline setup/symfony_server.rst]: You have used triple ticks here twice :)

File: setup/symfony_server.rst [hunk 5]
```diff
@@ -357,6 +357,25 @@ When Docker services are running, browse a page of your Symfony application and
...
+You start your containers like this:
+
+.. code-block:: bash
+
+    COMPOSE_FILE=docker/docker-compose.yaml COMPOSE_PROJECT_NAME=project_name docker-compose up -d
+
+And if you are using the same env variables with the Symfony CLI, things will work. For instance:
```
Comment (fabpot) [inline setup/symfony_server.rst]: environment instead of env?

File: setup/symfony_server.rst (hunk 4 above)
Comment (Plopix) [reply setup/symfony_server.rst]: ...sorry

File: setup/symfony_server.rst (hunk 5 above)
Comment (Plopix) [reply setup/symfony_server.rst]: done

File: setup/symfony_server.rst [hunk 6]
```diff
@@ -357,6 +357,25 @@ When Docker services are running, browse a page of your Symfony application and
...
+
+    COMPOSE_FILE=docker/docker-compose.yaml COMPOSE_PROJECT_NAME=project_name docker-compose up -d
+
+And if you are using the same environment variables with the Symfony CLI, things will work. For instance:
+
+.. code-block:: bash
+
+    COMPOSE_FILE=docker/docker-compose.yaml COMPOSE_PROJECT_NAME=project_name symfony var:export
+
+If you have more than one docker-compose files you can provide them all separated by `:`
```
Comment (Plopix) [inline setup/symfony_server.rst]: should it be double ticks too here?

Comment (Plopix) [reply setup/symfony_server.rst]: I did, I think I had to.

**Review by maxhelias — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Sébastien thanks for this contribution! (and thanks to reviewers too).

Please note that we reword some bits while merging to make the explanation a bit more concise. Thanks!

**Pull request merged**